app/views/pages/medicaid/intro_household_member.erb

```diff
@@ -0,0 +1,32 @@
+<h4 class="step-header step-header__title--centered">Introduction</h4>
+
+<div class="form-card">
+  <header class="form-card__header">
+    <div class="form-card__title">Add a member of your household</div>
+  </header>
+
+  <div>
+    <label class="form-question" for="example_text">What is their first name?</label>
+    <input type="text" class="text-input" name="example_text">
+
+    <label class="form-question" for="example_text">What is their last name?</label>
+    <input type="text" class="text-input"  name="example_text">
+
+    <label class="form-question" for="example_text">What is their gender?</label>
```

Comment:
didn't we want some helper text on this re: "Gender on license" or something? I know all of us would love to make this non-binary but we need to collect the info the gov't wants

Reply:
💯  @alanjosephwilliams do you have some good copy/helper text for this question?